changes/2517-peterroelants.md

```diff
@@ -0,0 +1,2 @@
+- Documentation update how to custom compile pydantic when using pip install.
+- Small change in setup.py to allow for custom CFLAGS when compiling.
```

Comment:
```suggestion
Documentation update how to custom compile pydantic when using pip install, small change in `setup.py` to allow for custom CFLAGS when compiling.
```

I'm pretty sure adding multiple bullet points here would break (or confuse) [`make_history.py`](https://github.com/samuelcolvin/pydantic/blob/master/changes/make_history.py).